packages/extension-chakra-storefront/src/components/header/index.test.js

```diff
@@ -59,13 +59,13 @@ beforeEach(() => {
 afterEach(() => {
     localStorage.clear()
 })
-test.skip('renders Header', async () => {
+test('renders Header', async () => {
     renderWithProviders(<Header />)
 
     await waitFor(() => {
         const menu = screen.getByLabelText('Menu')
         const logo = screen.getByLabelText('Logo')
-        const account = screen.getByLabelText('My Account')
+        const account = screen.getByLabelText(/my account/i)
```

Comment:
The header tests were not passing because of case sensitivity: expecting "My Account" vs receiving "My account".